src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -142,43 +142,43 @@
 ;;; Public
 
 (schema/defn ^:always-validate
-  initialize-config :- jruby-schemas/JRubyPuppetConfig
+  initialize-config :- jruby-schemas/JRubyConfig
   [config :- {schema/Keyword schema/Any}]
-  (-> (get-in config [:jruby-puppet])
+  (-> (get-in config [:jruby])
```

Comment:
Hmmm... now that I think about it... if we end up getting rid of the TK service, then we won't be reading any config automagically, we'll just have functions where the caller needs to pass us the config map, and maybe it's not unreasonable to expect Puppet Server to transform that map before passing it to us.  So maybe the way you have this right now is the best thing to do for the moment, and we can revisit after we figure out the Puppet Server integration a little more?